docs/addons/argo-cd.md

```diff
@@ -138,9 +179,28 @@ The framework provides support to supply repository and administrator secrets in
 2. Replicate the secret to all the desired regions.
 3. Set the secret name in `adminPasswordSecretName` in ArgoCD add-on configuration.
 
+Alternatively to get started, the admin password hash can be set bypassing the AWS Secret by setting the following structure in the values properties of the add-on parameters:
+
+```typescript
+import * as bcrypt from "bcrypt";
+
+.addOns(new ssp.addons.ArgoCDAddOn({
+         ... // other settings
+         values: {
+           "configs": {
+                "secret": {
+                    "argocdServerAdminPassword": bcrypt.hash(<your password plain text>, 10) // or just supply <your bcrypt hash> directly
```

Comment:
@parkand1 you can use this for the workshop. this won't require extra privileges and will work through the pipeline with no codebuild role change.